views/trials-details.html

```diff
@@ -151,31 +151,22 @@ <h2 class="disabled">{{ name }}</h2>
     <h2>Identifiers</h2>
     <dl class="id-list">
       <dt>
-        OpenTrials ID:
+        OpenTrials:
       </dt>
       <dd>
         {{ trial.id }}
       </dd>
 
-      {% if primaryId %}
+      {% if trial.identifiers %}
       <dt>
-        Primary ID:
+        Related:
```

Comment:
Ok here instead of `related` should be: 
ClinicalTrials.gov: 
<id>
Euctr:
<id>
etc